requirements-ext.txt

```diff
@@ -1 +1,2 @@
 numpy>=1.6.1
+singledispatch
```

Comment:
This was previously a requirement, except we always installed in the context where single dispatch was already available I guess?